packages/@aws-cdk/aws-apigatewayv2/lib/http/stage.ts

```diff
@@ -37,6 +45,21 @@ export interface HttpStageProps extends HttpStageOptions {
   readonly httpApi: IHttpApi;
 }
 
+/**
+ * Access logging configuration
+ */
+export interface AccessLogSettings {
+  /**
+   * CloudWatch Logs log group to receive access logs
+   */
+  readonly destination: ILogGroup;
+
+  /**
+   * A single line format of the access logs of data, as specified by selected $context variables. The format must include at least $context.requestId.
+   */
+  readonly format: string;
```

Comment:
Take a look at how access logging is set up in `aws-apigateway` module and use the same structure.